src/Jackalope/Transport/DoctrineDBAL/CachedClient.php

```diff
@@ -242,7 +251,7 @@ public function getNode($path)
 
         if (false !== ($result = $this->caches['nodes']->fetch($cacheKey))) {
             if ('ItemNotFoundException' === $result) {
-                throw new ItemNotFoundException(sprintf('Item "%s" not found in workspace "%s"', $path, $this->workspaceName));
+                throw new ItemNotFoundException("Item '$path' not found in workspace '$this->workspaceName'");
```

Comment:
Ok, although I like string inlining because it's shorter and I read article about string inlining for micro optimization: https://blog.blackfire.io/php-7-performance-improvements-encapsed-strings-optimization.html :-D :-D 

I will revert this change :-)